launcher/commands/check.py

```diff
@@ -52,8 +60,44 @@ def run(self, args) -> None:
 
             if md5 != info['MD5 Hash']:
                 errors += [f"Error: {file.name} -- remote({info['MD5 Hash']}) != local({md5})"]
```

Comment:
Move this line in the else statement (L71)

Reply:
Line moved.